### DISCUSSION TITLE
[Question]: How to use HTTP API via SSL and authentication (cURL)?

### DISCUSSION THREAD

**Discussion by christianschneider89** [blakeblackshear/frigate#13294] [Ask A Question] [answered]
### What is your question?

Maybe I missed something in the documentation, but how can I use the Frigate HTTP API with cURL via SSL and with authentication?

I tried to use the API with cURL like this:
curl -u apiuser:vYHSiqCdRMQ4eVh4P-HRxZ -k https://192.168.0.18:8971/api/config

I always get:

```
<html>
<head><title>401 Authorization Required</title></head>
<body>
<center><h1>401 Authorization Required</h1></center>
<hr><center>nginx/1.25.3</center>
</body>
</html>
```

**Comment by NickM-27** [comment, marked as answer]
you need to use the login api and then attach the token to the request

**Reply by christianschneider89** [reply]
So, with one cURL request it's impossible to do the job?

**Reply by NickM-27** [reply]
currently, yes

**Reply by christianschneider89** [reply]
Ok, thank you.
I just created a feature request: https://github.com/blakeblackshear/frigate/issues/13297
